superset/translations/ru/LC_MESSAGES/messages.json

```
@@ -4,60 +4,80 @@
     "superset": {
       "": {
         "domain": "superset",
-        "plural_forms": "nplurals=1; plural=0",
+        "plural_forms": "nplurals=1; plural=0;",
         "lang": "ru"
       },
-      "Home": [""],
+      "Home": ["Главная"],
       "Annotation Layers": ["Слои аннотаций"],
       "Manage": ["Управление"],
       "Databases": ["Базы данных"],
       "Data": ["БД"],
-      "Datasets": ["Базы данных"],
+      "Datasets": ["Датасеты"],
       "Charts": ["Графики"],
-      "Dashboards": ["Отчёты"],
-      "Plugins": [""],
+      "Dashboards": ["Дашборды"],
+      "Plugins": ["Плагины"],
       "CSS Templates": ["Шаблоны CSS"],
-      "Row level security": [""],
+      "Row level security": ["Безопасность на уровне строк"],
       "Security": ["Безопасность"],
-      "Import Dashboards": ["Импорт Отчетов"],
+      "Import Dashboards": ["Импорт дашбордов"],
       "SQL Editor": ["Редактор SQL"],
-      "SQL Lab": ["Лаборатория"],
+      "SQL Lab": ["Лаборатория SQL"],
       "Saved Queries": ["Сохраненные запросы"],
       "Query History": ["История запросов"],
       "Upload a CSV": ["Загрузить CSV"],
-      "Upload Excel": [""],
+      "Upload Excel": ["Загрузить файл Excel"],
       "Action Log": ["Журнал Действий"],
-      "Dashboard Emails": ["Отчёты"],
-      "Chart Email Schedules": [""],
-      "Alerts": [""],
-      "Alerts & Reports": [""],
+      "Dashboard Emails": ["Рассылка дашбордов"],
+      "Chart Email Schedules": ["Рассылка графиков"],
+      "Alerts": ["Предупреждения"],
```

I would suggest the "Оповещения" option
```suggestion
      "Alerts": ["Оповещения"],
```

---

"Оповещения" seems more like notifications for me. Don't you think?

---

I think "Оповещение" is more relevant for that functionality. Developers decided to call Alert - but in fact it's notification.

---

Okay, thank you, guys. Fixed translation of 'Alerts' through all messages.